components/ButtonBar/src/MDCButtonBar.h

```diff
@@ -98,6 +98,17 @@ IB_DESIGNABLE
  */
 @property(nonatomic, copy, nullable) NSArray<UIBarButtonItem *> *items;
 
+/**
+ Returns the rect of the item's view within the coordinate space of @c view.
+
+ If the provided item is not contained in @c items, then the behavior is undefined.
+
+ @param item The item within @c items whose rect should be computed.
+ @param coordinateSpace The coordinate space the returned rect should be in relation to.
+ */
+- (CGRect)rectForItem:(nonnull UIBarButtonItem *)item
```

Comment:
Should this be `boundsForItem:...`

Reply:
This naming matches UIKit's convention in the `convertRect:toCoordinateSpace:` family of APIs that this API is modeled after.